dotCMS/src/main/java/com/dotmarketing/factories/MultiTreeAPIImpl.java

```diff
@@ -905,6 +906,8 @@ private boolean doesPageContentsHaveContainer(
 
         if(pageContents.contains(container.getIdentifier(), containerUUID.getUUID())){
             return true;
+        } else if(pageContents.contains(container.getIdentifier(), ParseContainer.PARSE_CONTAINER_UUID_PREFIX + containerUUID.getUUID())) {
+            return true;
```

Comment:
test?

Reply:
Agree. We need a test here since it's a change at API level.